lib/importer/templates/select_footer_row.html

```diff
@@ -1,19 +1,29 @@
 {% extends "layouts/main.html" %}
 
+{% block head %}
+{{ super() }}
+{% endblock %}
+
 {% block pageTitle %} – {{ serviceName }} – GOV.UK Prototype Kit {% endblock %}
```

Comment:
So is the user meant to add their own page title before the dash?

Reply:
Nope, just didn't notice.